presubmit.py

```diff
@@ -210,6 +210,15 @@ def lint(paths: List[Path]) -> bool:
 def pytype(paths: List[Path]) -> bool:
     """Run pytype on |path| if it is a python file. Return False if it fails
     type checking."""
+    # Pytype isn't supported on Python3.8+. See
+    # https://github.com/google/pytype/issues/440.
+    assert sys.version_info.major == 3
```

Comment:
Shouldn't this assert be in main(), we shouldn't run presubmit if we are not on version 3, also add a message at end
assert sys.version_info.major == 3, "Needs python 3"

Reply:
This assertion is purely for show so that the check for ".7" doesn't look weird. As I mentioned, until Python 4 comes around there's no danger of using the wrong Python major version since this file is syntactically invalid under Python2.